demo/src/app/extensions/responsive-extension.component.html

```diff
@@ -1,6 +1,5 @@
 <ng-template #preview>
   <table datatable [dtOptions]="dtOptions" class="row-border hover"></table>
 </ng-template>
-<app-base-demo [pageTitle]="pageTitle" [mdIntro]="mdIntro" [mdInstall]="mdInstall" [mdHTML]="mdHTML" [mdTS]="mdTS"
-  [template]="preview">
+<app-base-demo [pageTitle]="pageTitle" [mdIntro]="mdIntro" [mdInstall]="mdInstall" [mdHTML]="mdHTML" [mdTS]="mdTS" mdTSV2="mdTSV2" [template]="preview">
```

Comment:
```suggestion
<app-base-demo [pageTitle]="pageTitle" [mdIntro]="mdIntro" [mdInstall]="mdInstall" [mdHTML]="mdHTML" [mdTS]="mdTS" [mdTSV2]="mdTSV2" [template]="preview">
```

__issue__: Missing brackets `[]`.

Reply:
Fixed :)